src/main/java/com/glisco/isometricrenders/widget/DynamicLabelComponent.java

```diff
@@ -32,13 +32,15 @@ public DynamicLabelComponent shadow(boolean shadow) {
     }
 
     @Override
-    protected void applyHorizontalContentSizing(Sizing sizing) {
+    protected int determineHorizontalContentSize(Sizing sizing) {
         this.width = 100;
+        return 100;
     }
 
     @Override
-    protected void applyVerticalContentSizing(Sizing sizing) {
+    protected int determineVerticalContentSize(Sizing sizing) {
         this.height = this.textRenderer.fontHeight;
```

Comment:
Same goes for this one, return the value directly instead of first applying it to the field